src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl

```diff
@@ -13297,6 +13312,14 @@ unit:HR-PER-YR
   qudt:applicableSystem sou:SI ;
   qudt:applicableSystem sou:USCS ;
   qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D1 ;
+    # MISSING MULTIPLIER: unit:HR-PER-YR - calculated from factors: 0.0001140771161305042208532968286561716
```

Comment:
As implied by the URI, this is a confusing unit, because there is a fixed value for the number of hours in a year. The description clarifies the intention however. I'm thinking that we should deprecate this unit in favor of a renamed unit, HR_Activity-PER-YR.

Thinking some more, I think it is cleaner to keep the unit the same, but have it refer to one or more quantity kinds, to include quantitykind:TimeRatio_Activity or quantitykind:ActivityTimeRatio, with a skos:broader to quantitykind:TimeRatio.

Thoughts?

Reply:
I don't see the problem. How is it different from MilliM-PER-M?

Reply:
I think the difference is that MilliM-PER-M references a quantity kind that gives it a context (in this case, a gradient). It is the lack of a context, for two units of the same dimension but different magnitude, that threw me. Maybe it's just the side of the bed I woke up on today.

Reply:
I don't understand the request and whether you want that new QK now or not, but since you approved the PR, I think we can merge (thus avoid collecting more merge conflicts) and add this QK later if we still think it is needed.

Reply:
I want the new QK but I'm fine with waiting for our next Board call to discuss.